docs/tutorial/fdfitting.rst

```diff
@@ -131,25 +113,23 @@ If we also expect an offset in the dependent parameter, we can simply add an off
     Name            Value  Unit      Fitted      Lower bound    Upper bound
     ------------  -------  --------  --------  -------------  -------------
     DNA/d_offset     0.01  [au]      True               -0.1            0.1
-    DNA/Lp          40     [nm]      True                0            100
-    DNA/Lc          16     [micron]  True                0            inf
-    DNA/St        1500     [pN]      True                0            inf
-    kT               4.11  [pN*nm]   False               0              8
+    DNA/Lp          40     [nm]      True              0.001            100
+    DNA/Lc          16     [micron]  True            0.00034            inf
+    DNA/St        1500     [pN]      True                  1            inf
+    kT               4.11  [pN*nm]   False              3.77              8
     DNA/f_offset     0.01  [pN]      True               -0.1            0.1
 
-From the above example, you can see how easy it is to composite models. Sometimes, models become more complicated. For
-instance, we may have two worm like chain models that we wish to add, and then invert. For the Odijk model, this can be
+Sometimes, models become more complicated. For instance, we may have two worm like chain models that we wish to add, and then invert. For the Odijk model, this can be
 done as follows::
 
     model = lk.ewlc_odijk_distance("DNA") + lk.ewlc_odijk_distance("protein") + lk.distance_offset("offset")
     model = model.invert()
 
 Note how we added three models and then inverted the composition of those models. Models inverted via `invert()` will
```

Comment:
Now that we have API docs, it would be nice to link the docs page for `.invert()` here.
```suggestion
Note how we added three models and then inverted the composition of those models. Models inverted via :meth:`.invert() <lumicks.pylake.fitting.model.Model.invert()>` will
```